### DISCUSSION TITLE
My school provided email displays the wrong school.

### DISCUSSION THREAD

**Discussion by PentiumY** [community/community#17802] [GitHub Education]
Alright so I recently tried to apply for the student program but when I entered my school email it displayed the wrong school. I’ve contacted github support but their bot automatically closes my tickets. So I’ll ask here instead. Tl;dr my school provided email is linked to the wrong school and I can’t prove I go there since I don’t.

**Comment by juanpflores** [comment]
Do you mind sharing with us the ticket number so our team can give it a review?
